### DISCUSSION TITLE
Issues with projects

### DISCUSSION THREAD

**Discussion by paulGeoghegan** [community/community#68005] [Accessibility] [answered]
### Select Topic Area

Bug

### Body

Hello, hope you’re doing well. I have recently started using Github projects and have noticed numerous accessibility issues. The actual project view is not very screen reader friendly. However, this could be easily addressed in the following ways:
1. Make each section title (e.g. done) a heading or at least add the role for a heading to them. This means that screen reader users could quickly navigate between each section in their projects, where as currently it can be a nightmare to navigate.
2. Each issue should either be one large clickable element using the button role or each issue title should be a heading one level lower than the section heading to allow screen reader users to better make use of each issue.
There also issues with the modals that appear after you select an issue and when you change a value on the issue. It will often move the users focus away from the modal and back to the main body of the page. It is possible to get back to the modal, but this is not ideal. There is also an issue with inputs that appear as popups over the modal, where after selecting a value you cannot leave the popup and you can not manually scroll through the list of options. Also, when searching, sometimes you cannot select the suggestions presented to you.
For reference these issues were found using Firefox and NVDA both on their latest versions, but I suspect that neither of these tools are the reason these issues are occurring.
Obviously, there are other possible solutions to these problems. The ones above are just examples that work for me.

**Comment by georgetoloraia** [comment]
@paulGeoghegan 

# These insights are incredibly useful for web developers and product managers at GitHub. For actionable steps, you might consider:

## Reporting the Issues:
- If you haven't already, report these issues directly to GitHub. They have a platform for reporting bugs and a forum for suggesting features.

## Community Engagement:
- Sharing your findings with the community (e.g., on forums, in GitHub discussions) can also be helpful. It raises awareness and might prompt quicker action from the developers.

**Reply by ghostinhershell** [reply]
Hi @georgetoloraia, 

We’ve clarified our stance on using generative AI tools like ChatGPT within our Community via [this announcement](https://github.com/orgs/community/discussions/69006). Please review the guidelines to ensure your post meets them as failure to adhere to those rules can result in action taken by our moderator team.  You can read our updated [Code of Conduct](https://github.com/community/community/blob/baa5cd9020ef3f2bbf86dca1191d9c90f9b0822d/CODE_OF_CONDUCT.md#reasonable-use-of-ai-generated-content) and the [announcement](https://github.com/orgs/community/discussions/69006) for more details. Thank you for helping us maintain an authentic and beneficial space for everyone.

**Comment by ghostinhershell** [comment]
Hi @paulGeoghegan, 

Thanks so much for reaching out in the Accessibility community. I see that you originally shared this feedback back in September. By any chance, has anything changed about the observations that you've shared here since then?

**Reply by paulGeoghegan** [reply]
No it hasn't

**Reply by ghostinhershell** [reply]
Gotcha, I'll submit your feedback to the Accessibility team so that it's on their radar. Thanks so much for contributing to our community!

**Comment by ericwbailey** [comment, marked as answer]
Hi @paulGeoghegan. I'm a member of the GitHub accessibility team, and I've been assigned to help out with this issue. Project swimlanes and the title of the cards now have `h2` and `h3` elements assigned to them, respectively. 

You may need to refresh your browser tab for these changes to take effect. 

I'll also be evaluating the other concerns you mentioned, as well.

**Comment by paulGeoghegan** [comment]
This may not be very professional but I literally just screamed!This has actually made my day. I will say it did take the best part of a year but I now love projects!Before this I was getting my wife to manage the GitHub issues but now I can actually navigate them by myself!It means a lot that I can now be that bit more independent if I want so thank you again.I would honestly love to help out more with giving accessibility feedback for GitHub if I can. Paul Geoghegan From: Eric BaileySent: Wednesday 21 August 2024 21:52To: community/communityCc: Paul Geoghegan; MentionSubject: Re: [community/community] Issues with projects (Discussion #68005) Hi @paulGeoghegan. I'm a member of the GitHub accessibility team, and I've been assigned to help out with this issue. Project swimlanes and the title of the cards now have h2 and h3 elements assigned to them, respectively.You may need to refresh your browser tab for these changes to take effect.I'll also be evaluating the other concerns you mentioned, as well.—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you were mentioned.Message ID: ***@***.***>

**Reply by ericwbailey** [reply]
@paulGeoghegan Thank yo so much for sharing that with me. I'm really glad I was able to help you use GitHub.

I'd also like to take you up on your offer to help, if you're still interested. The GitHub accessibility team is working to involve the perspective of daily assistive technology users earlier in our design and development processes. Part of that has been forming a customer research panel, where we can get direct feedback on what is and is not working for new features and functionality we're working on.

Here is [a sign up form](https://forms.gle/23Jgw5tRwPLZHNTs5), if you're interested in participating. There is absolutely no obligation to do so, so only fill it out if it's something you feel comfortable doing.

I also wanted to reiterate that the other issues you reported are things we are working to address, and have been captured in our internal tracking and triaging systems. I hope we can address them sooner than later.

**Comment by paulGeoghegan** [comment]
I just signed up so thanks so much for the link.
I really do appreciate your work.

**Reply by ericwbailey** [reply]
That's fantastic news! I hope we get to chat one day soon 🙂
